CREDITS.md

```diff
@@ -1330,3 +1330,28 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 ```
+
+### https://github.com/guille/ms.js
+#### client/lib/wp/sync-handler
```

Comment:
Do we gain any benefit from pointing out usages? This will get out of date pretty quickly.